### ISSUE TITLE
Tests for twisted.application should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7713]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7713|
|Type|enhancement|
|Created|2014-11-13 15:27:40Z|
|Branch|https://github.com/twisted/twisted/tree/application-tests-names-7713|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7713.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/836/8364d6f0dccb83edda78ee89cabc6d9eb22ce3d2/237dcc26bdbb61ae54c1e9652a0b6ce6de31ba64.patch) (1264 bytes) - added by eeshangarg on 2014-11-13 15:32:08Z - 
* [bugfix_7713_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/836/8364d6f0dccb83edda78ee89cabc6d9eb22ce3d2/324f70c685ffa9abb526f8596b2bbec623655e51.patch) (1366 bytes) - added by eeshangarg on 2014-11-17 11:54:38Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7713 7713
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_application_tests_names_7713 branches/application-tests-names-7713
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1415892460413029 1415892460413029
changetime__1417023323941318 1417023323941318
version__None None
owner__exarkun exarkun
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg set owner to @glyph|
|-|-|

Hi! I just added a patch that changes all of the tests in twisted.application to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks!
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph removed owner|
|-|-|

Un-assigning all review tickets assigned to me.  Unless there's a specific reason _I_ need to review something, let's leave them unassigned :-).

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

Looks good. Thanks!

To have it merged it will need a mandatory news .misc file.

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_4"></a>|@eeshangarg commented|
|-|-|

Hello!

I just added a new patch called bugfix_7713_2.patch that includes the .misc news file. Thanks a lot for the review. :)

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [43506]) Branching to 'application-tests-names-7713'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [43507]) Apply bugfix_7713_2.patch 

Refs [#7713](https://github.com/twisted/twisted/issues/7713)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Thanks.  I fixed one more missed renaming in r43508.  The rest of this looks good to me.  I'll merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set status to `closed`|
|-|-|

(In [43509]) Merge application-tests-names-7713

Author: eeshangarg
Reviewer: adiroiban, exarkun
Fixes: [#7713](https://github.com/twisted/twisted/issues/7713)

Rename all of the test case classes in `twisted.application.test` to conform to the
recently documented test case class naming convention.
